Framework/Algorithms/test/Stitch1DManyTest.h

```diff
@@ -993,8 +1175,11 @@ class Stitch1DManyTest : public CxxTest::TestSuite {
     }
 
     TS_ASSERT_EQUALS(histNames[0], "CreateWorkspace");
```

Comment:
Could we have
```
const std::string createWsName = "CreateWorkspace";
const std::string groupWsName = "GroupWorkspaces";

TS_ASSERT_EQUALS(histNames[1], createWsName); ...etc.
```
This way if the names change we only have to change in in one place instead of 4